app/controllers/idv/otp_verification_controller.rb

```diff
@@ -13,12 +14,12 @@ class OtpVerificationController < ApplicationController
     def show
       # memoize the form so the ivar is available to the view
       phone_confirmation_otp_verification_form
-      analytics.idv_phone_confirmation_otp_visit
+      idv_analytics.idv_phone_confirmation_otp_visit
```

Comment:
Yeah I agree with the concern this could be easy to overlook for future events. The regression specs give us some assurances, but still pretty error-prone.

By "keyword", do you mean the "IdV" prefix in the method or event name?

I see there's some prior art for decorating contextual analytics details in [`Analytics#track_event`](https://github.com/18F/identity-idp/blob/e170866f6dbc14f707ce63b2920551f316af96fd/app/services/analytics.rb#L14-L43) that we could extend for IdV events to simplify this. I'm not a huge fan of baking that into the Analytics class, but it would simplify the code quite a bit and help ensure consistency.

Or maybe the concern could override the default `analytics` method and add the IdV-specific code as a decorated wrapper of `Analytics#track_event`. 🤔

Reply:
> By "keyword", do you mean the "IdV" prefix in the method or event name?

no sorry I meant the parameter (ex if a method has a `proofing_components:` in its signature)

> Or maybe the concern could override the default `analytics` method and add the IdV-specific code as a decorated wrapper of `Analytics#track_event`. 🤔

Yes that is more like what I was thinking. Including the concern/mixin would override `analytics` for just that controller

Reply:
Updated in 5b32740 to override base analytics method.